src/components/ItemEditorPage/RightPanel/Tags/Tags.tsx

```diff
@@ -65,7 +65,7 @@ export default class Tags extends React.PureComponent<Props, State> {
       <div className={`Tags ${value.length > 0 ? '' : 'blank'} ${isDisabled ? 'is-disabled' : ''}`.trim()}>
         <div className="values">
           {value.map(value => (
-            <div className="value">
+            <div className="value" key={value}>
```

Comment:
Are tags unique? if not maybe we should use the `index` here

Reply:
same as above, both tags and multiselect can't have the same value repeated